components/style/custom_properties.rs

```diff
@@ -513,8 +547,7 @@ impl<'a> CustomPropertiesBuilder<'a> {
                 CSSWideKeyword::Initial => {
                     map.remove(name);
                 }
-                CSSWideKeyword::Unset | // Custom properties are inherited by default.
-                CSSWideKeyword::Inherit => {} // The inherited value is what we already have.
+                _ => unreachable!(),
```

Comment:
Maybe worth being explicit with the match arms? (Seems unlikely that we'd add a wide keyword in the future, but...)

Reply:
Yes, fair point. (Well, there is `revert`...)